types/humanparser/index.d.ts

```diff
@@ -5,7 +5,7 @@
 
 declare namespace humanparser {
     interface NameOutput {
-        firstName: string;
+        firstName?: string | undefined;
```

Comment:
According to tests here: https://github.com/ralyodio/humanparser/blob/master/test/index.js#L214 This should be `string` if I'm not mistaken

Reply:
Yep I think that's what we all expect, but it appears that in cases where just a title + last name are passed in `firstName` is not present in the result and this looks intentional according to this [unit test](https://github.com/ralyodio/humanparser/blob/master/test/index.js#L175-L181)

Reply:
Thank you - did not expect to find that, after seeing the test for empty string

Reply:
Yeah it's pretty weird, potentially a bug that I could patch however the package doesn't look that active so for now worth just getting the types aligned to the current response. We saw a bunch of failures in prod due to this overnight 😅